src/oneD/StFlow.cpp

```diff
@@ -147,6 +147,13 @@ void StFlow::setTransport(Transport& trans)
     m_trans = &trans;
     m_do_multicomponent = (m_trans->transportType() == "Multi");
 
+    if (!m_do_multicomponent && m_do_soret) {
+        throw CanteraError("setTransport",
+                           "Thermal diffusion (the Soret effect) is "
```

Comment:
Extremely minor: with longer error messages, I tend to indent subsequent lines just one level, rather than lining up after the `(`, which gets to be a bit ridiculous once you're already 30 columns in.

Reply:
:+1: